app/config/routing.yml

```diff
@@ -7,6 +7,11 @@
 app:
     resource: @AppBundle/Controller/
     type:     annotation
+    prefix:   /{_locale}
+    requirements: 
+        _locale: en|fr|de|es|cs|ru|uk| 
```

Comment:
Actually, it should go into a `parameters` key in `config.yml` in that case (since it doesn't change on a machine-by-machine basis). I think that's a good idea (we'll add a comment explaining it)